### ISSUE TITLE
get_user_model does not exist in Django 1.4

### ISSUE THREAD

**Issue by wolever** [marcgibbons/django-rest-framework-docs#22]
The README claims that Django 1.4 is supported, but `docs.py` imports `get_user_model`, which doesn't exist in 1.4.

**Comment by marcgibbons** [comment]
You're right. This was added here. Wondering if we can do without a user instance... 
https://github.com/marcgibbons/django-rest-framework-docs/commit/74f8781450962cd56631264a729c8ab60cd8cb77

**Comment by g-m-a** [comment]
any fixes for this?

**Comment by benspaulding** [comment]
@marcgibbons I recognize that this project is no longer supported, so I don’t _expect_ you to take time to merge my commit, but it is there if you want it!
